Sources/Document/Time/TimeTicket.swift

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright 2022 The Yorkie Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+
+extension UInt64 {
+    func toString() -> String {
+        return String(self)
+    }
+}
+
+/**
+ * `TimeTicket` is a timestamp of the logical clock. Ticket is immutable.
+ * It is created by `ChangeID`.
+ */
+class TimeTicket {
+    private enum InitialValue {
+        static let initialDelimiter: UInt32 = 0
+        static let maxDelemiter: UInt32 = 4_294_967_295
```

Comment:
Can we use a constant here? 🤔

```suggestion
        static let maxDelemiter: UInt32 = Uint32.max
```